<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QWidget" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="MAIN">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>761</width>
     <height>681</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="MAIN_ChatLayout" stretch="0">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="Column">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>10</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(16,19,23,255);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="TopBar_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgba(34,35,39,255);</string>
         </property>
         <layout class="QHBoxLayout" name="TopBar">
          <property name="leftMargin">
           <number>35</number>
          </property>
          <property name="rightMargin">
           <number>35</number>
          </property>
          <item alignment="Qt::AlignLeft">
           <widget class="QFrame" name="UserIconAndNickInChat">
            <layout class="QHBoxLayout" name="LogoAndNickName">
             <property name="spacing">
              <number>25</number>
             </property>
             <item>
              <widget class="QPushButton" name="UsersLogoinChat">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:purple;
width:50px;
height:50px;
border-radius:25px;
color:White;
font-size:25px;
text-align:center;
</string>
               </property>
               <property name="text">
                <string>G</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="UsersNickInChat">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font-size:26px;
color:white;</string>
               </property>
               <property name="text">
                <string>Group-1</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="CallAndInfoButtons">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>20</number>
             </property>
             <item>
              <widget class="QPushButton" name="CallButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>45</width>
                 <height>45</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:rgba(38,40,45,255);
border:none;
border-radius:15%;
border:3px solid white;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../res-rec.qrc">
                 <normaloff>:/icons/icon/call_46dp_FFFFFF_FILL0_wght400_GRAD0_opsz48.svg</normaloff>:/icons/icon/call_46dp_FFFFFF_FILL0_wght400_GRAD0_opsz48.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="InfoButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>45</width>
                 <height>45</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:r#BB271A;
border:none;
border-radius:15%;
border:3px solid #BB271A;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../res-rec.qrc">
                 <normaloff>:/icons/icon/person_remove_46dp_BB271A_FILL0_wght400_GRAD0_opsz48.svg</normaloff>:/icons/icon/person_remove_46dp_BB271A_FILL0_wght400_GRAD0_opsz48.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="ChatLayout">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="Call">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame{
background-color:rgba(38,40,45,255);
border-top:3px solid white;
border-bottom:3px solid white;
}

QPushButton {
	border:3px solid white;
	border-radius:50%;
	color:white;
	font-size:30px;
}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignHCenter">
              <widget class="QFrame" name="horizontalFrame">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">QFrame {
border:none;
}

QPushButton {
	border:3px solid orange;
	border-radius:15%;
	color:white;
	font-size:30px;
}</string>
               </property>
               <layout class="QHBoxLayout" name="wrapperForButtons">
                <property name="spacing">
                 <number>25</number>
                </property>
                <item>
                 <widget class="QPushButton" name="muteMic">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">
border:2px solid white;
background-color:black;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../res-rec.qrc">
                    <normaloff>:/icons/icon/mic_off_28dp_FFFFFF_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/icon/mic_off_28dp_FFFFFF_FILL0_wght400_GRAD0_opsz24.svg</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>28</width>
                    <height>28</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="muteHeadphones">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border:2px solid white;
background-color:black;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../res-rec.qrc">
                    <normaloff>:/icons/icon/headset_off_28dp_FFFFFF_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/icon/headset_off_28dp_FFFFFF_FILL0_wght400_GRAD0_opsz24.svg</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>28</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="flat">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignVCenter">
                 <widget class="QPushButton" name="leaveCall">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border-color:#BB271A;
background-color:#BB271A;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../res-rec.qrc">
                    <normaloff>:/icons/icon/call_end_28dp_FFFFFF_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/icon/call_end_28dp_FFFFFF_FILL0_wght400_GRAD0_opsz24.svg</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>28</width>
                    <height>28</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="MessahesInChat">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="leftMargin">
              <number>45</number>
             </property>
             <property name="rightMargin">
              <number>45</number>
             </property>
             <item>
              <widget class="QListWidget" name="ChatScroll">
               <property name="styleSheet">
                <string notr="true">QListWidget {
border:none;
}


QListWidget::item
{
    background:none; 
}
QListWidget::item:selected
{
    background: none;
}

QScrollBar:vertical {
           border:none;
            width:10px;
             height:20px;
                                                            
  }
                                                        
 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical
 {
         heigth:0;
 }
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
 {
               background: none;
 }
  QScrollBar::add-line:vertical {
               height: 0px;
  }
 QScrollBar::sub-line:vertical {
 				height: 0px;
 }
QScrollBar::handle:vertical {
               background: orange;
                min-height:10px;
                 border-radius: 5px;
 }</string>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAsNeeded</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="ChatInputLayout">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="Chat_input">
             <property name="spacing">
              <number>7</number>
             </property>
             <item>
              <widget class="QLineEdit" name="Chat_input_">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit {
 width:250px;
height:30px;
border: 2px solid #323338;
border-radius:20%;
background-repeat: no-repeat;
background-position: left;
background-color:#323338;
font-size:16px;
color:#808994;
text-align:left;
padding-left:15px;
}

QPushButton:hover {
background-color:#656c76;
color:black;
background-image: url(:/icons/icon/search_24dp_000000_FILL0_wght400_GRAD0_opsz24.svg); 
background-repeat: no-repeat;
background-position: left;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Введите сообщение</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Attcahment_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:rgba(38,40,45,255);
border-radius:25%;
border:3px solid white;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../res-rec.qrc">
                 <normaloff>:/icons/icon/attach_file_46dp_FFFFFF_FILL0_wght400_GRAD0_opsz48.svg</normaloff>:/icons/icon/attach_file_46dp_FFFFFF_FILL0_wght400_GRAD0_opsz48.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Send_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:rgba(38,40,45,255);
border-radius:25%;
border:3px solid white;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../res-rec.qrc">
                 <normaloff>:/icons/icon/send_30dp_FFFFFF_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/icon/send_30dp_FFFFFF_FILL0_wght400_GRAD0_opsz24.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../res-rec.qrc"/>
 </resources>
 <connections/>
</ui>
